frontend/src/main/web/src/components/Flex/readme.md

```diff
@@ -0,0 +1,2 @@
+For a more comprehensive guide on how Flexbox works, check out
+[A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/).
```

Comment:
Flexbox in html is just a div component with css `display:flex`. This component is to help user that have no knowledge in css or flexbox with some common attribute.